ogticrd/covid-safe-paths
# User status toast

#### Description:
The user will now see a quick preview of its contact status in a little banner in the bottom of the homescreen when they open the app. The visuals are described in the images below.


#### Linked issues:
[Ticket 250](https://trello.com/c/kl7PQ1ar/250-incluir-el-un-flotante-donde-se-muestre-el-estado-del-usuario-en-el-app)


#### Screenshots:
![Simulator Screen Shot - iPhone 11 - 2020-09-24 at 18 48 21](https://user-images.githubusercontent.com/56127045/94208487-f6104680-fe97-11ea-8bdb-17050c96922f.png)
![Simulator Screen Shot - iPhone 11 - 2020-09-24 at 18 48 42](https://user-images.githubusercontent.com/56127045/94208493-f6a8dd00-fe97-11ea-97d8-b47de03c88d4.png)
![Simulator Screen Shot - iPhone 11 - 2020-09-24 at 18 52 11](https://user-images.githubusercontent.com/56127045/94208495-f6a8dd00-fe97-11ea-8292-c8643267b2cc.png)



#### How to test:
`git fetch && git checkout user-status-toast && yarn install` run the project
When you open homeScreen, your contact status should appear in a banner at the bottom of the screen for three seconds. To check if that's the right status you can go to LocationMatch screen. Also check turning off the location.
